src/Filters/Traits/AppliesFiltersTrait.php

```diff
@@ -36,10 +37,16 @@ protected function applyFiltersToQuery(array $filters, Builder $query): Builder
 
             // Append time to date, to query with <= and >= like Elasticsearch, instead of using DB::raw('DATE(column)')
             if (strpos($name, 'date_from') !== false || strpos($name, 'date_to') !== false) {
-                if (!preg_match('/^\d{4}-\d{2}-\d{2}$/', $value)) {
-                    throw new UnprocessableEntityException('The filter ' . $name . ' is not in ISO 8601 date format');
+                if (!preg_match('/^(\d{4}-\d{2}-\d{2})|(\d+)$/', $value)) {
+                    throw new UnprocessableEntityException('The filter ' . $name . ' is not a timestamp or in ISO 8601 date format.');
```

Comment:
It is very tempting to include full-scale ISO 8601 support right here, isn't it? I think Carbon can be used test again ISO 8601.

Reply:
That would have to then be in addition to the already existing date string?